console/react/test_data/qdrService.mock.js

```diff
@@ -37,19 +37,21 @@ export const mockService = ({ onSendMethod }) => {
   const cbSendMethod = onSendMethod ? onSendMethod : () => {};
   return {
     management: {
+      schema: () => schema,
       connection: {
         sendMethod: () => {
           cbSendMethod();
           return Promise.resolve(methodResults);
         },
         is_connected: () => true,
-        setReconnect: () => {}
+        setReconnect: () => {},
+        getReceiverAddress: () => 'mockReceiverAddress',
```

Comment:
Fix for `TypeError: this.service.management.connection.getReceiverAddress is not a function`

Reply:
Probably better to return 'amqp:/_topo/0/routerName/' instead of 'mockReceiverAddress'. The code that calls getReceiverAddress() will attempt to split the returned string on '/', add '$management' to the resulting array, and then contruct a router id from the array. 
However, that code would only be called if there was only 1 router in the network that is being tested. The qdrMockService.js file sets up 2 routers, so that code is never hit in tests that use qdrMockService.js anyway. But if that ever changes, it would be good to return an actual router ID.

Reply:
I'll change it per your suggestion.